examples/raw_client_file_example.py

```diff
@@ -0,0 +1,81 @@
+from odp.client import OdpClient
+from odp.client.dto.file_dto import FileMetadataDto
+from odp.client.exc import OdpFileAlreadyExistsError, OdpResourceExistsError, OdpValidationError
+from odp.dto import Metadata
+from odp.dto.catalog import DatasetDto, DatasetSpec
+from odp.dto.common.contact_info import ContactInfo
+
+# Instantiate the client without specifying a token provider.
+# The token provider will be set based on the environment.
+client = OdpClient()
+
+# Declare a dataset manifest to add to the catalog
+
+print("Creating sample dataset")
+
+dataset = DatasetDto(
+    metadata=Metadata(
+        name=client.personalize_name("sdk-raw-example"),
+        display_name="SDK Raw Example",
+        description="A test dataset for raw data",
+        labels={"hubocean.io/test": True},
+    ),
+    spec=DatasetSpec(
+        storage_controller="registry.hubocean.io/storageController/storage-raw-cdffs",
+        storage_class="registry.hubocean.io/storageClass/raw",
+        maintainer=ContactInfo(
+            contact="User McUsername <user.mcusername@emailprovider.com>",
+            organization="Organization Name",
+        ),
+    ),
+)
+
+# The dataset is created in the catalog.
+try:
+    dataset = client.catalog.create(dataset)
+    print("Resource created successfully:", dataset)
+except OdpValidationError as e:
+    print(e)
```

Comment:
Execution should not continue after this point. Remove exception-handling for `OdpValidationError` or raise another exception.

For example:

```python
try:
    dataset = client.catalog.create(dataset)
    print("Resource created successfully:", dataset)
except OdpValidationError as e:
    raise ValueError("Invalid dataset manifest") from e
except OdpResourceExistsError as e:
    ...
```